### PR TITLE
[#503] Centralize BotBuilder version - Python

### PR DESCRIPTION
**Pull request by ceciliaavila**
Addresses # 503

## Description
This PR adds a `requirement.txt` file in `Bots/Python` folder to be referenced in each bot's _requirement.txt_ having the packages and their versions centralized in one place.

## Specific Changes
- Added a `requirement.txt` in `Bots/Python` with the packages and versions used by the bots.
- Updated each bot's `requirement.txt` to reference the one in the root folder.
- Updated the `evaluateDependenciesVariables` script to replace the reference in the bot's _requirement.txt_ with the content of the root file.

## Testing
The following images show the pipelines passing using different combinations of version and feed.
image
image
image

### CODE REVIEW

File: build/yaml/deployBotResources/python/evaluateDependenciesVariables.yml [hunk 1]
```diff
@@ -60,10 +60,19 @@ steps:
             }
           }
 
-          # Add the $source source at the beginning of requirements
+          ## Update requirements.txt
```
Comment (sw-joelmut) [inline build/yaml/deployBotResources/python/evaluateDependenciesVariables.yml]: Extra `#`

File: build/yaml/deployBotResources/python/evaluateDependenciesVariables.yml [hunk 2]
```diff
@@ -60,10 +60,19 @@ steps:
...
           $file = "${{ parameters.source }}/requirements.txt"
           $content = @(Get-Content $file)
+          $rootFile = "Bots/Python/requirements.txt"
+
+          # replace root requirements.txt reference with file content
```
Comment (sw-joelmut) [inline build/yaml/deployBotResources/python/evaluateDependenciesVariables.yml]: First letter uppercase `replace` =>`Replace`

**Review by sw-joelmut — APPROVED** [review]
Just minor things

File: build/yaml/deployBotResources/python/evaluateDependenciesVariables.yml (hunk 1 above)
Comment (ceciliaavila) [reply build/yaml/deployBotResources/python/evaluateDependenciesVariables.yml]: Done. Thanks!

File: build/yaml/deployBotResources/python/evaluateDependenciesVariables.yml (hunk 2 above)
Comment (ceciliaavila) [reply build/yaml/deployBotResources/python/evaluateDependenciesVariables.yml]: Done, thanks!

**Pull request closed without merging**